CODEOWNERS

```diff
@@ -199,6 +199,17 @@ k8s/ithc/common/xui/webapp.yaml @hmcts/xui
 k8s/ithc/common/xui/manage-organisations.yaml @hmcts/xui
 k8s/ithc/common/xui/approve-org.yaml @hmcts/xui
 k8s/ithc/common/xui/share-case.yaml @hmcts/xui
+k8s/perftest/common/xui/xui-tc.yaml @hmcts/xui
+k8s/perftest/common/xui/webapp.yaml @hmcts/xui
+k8s/perftest/common/xui/manage-organisations.yaml @hmcts/xui
+k8s/perftest/common/xui/approve-org.yaml @hmcts/xui
+k8s/perftest/common/xui/share-case.yaml @hmcts/xui
+k8s/demo/common/xui/xui-tc.yaml @hmcts/xui
```

Comment:
Demo doesn't need code owners as of now.

Reply:
okay I will remove this